src/samples.jl

```diff
@@ -1,7 +1,7 @@
 function load_distributions_samples()
 return quote
 
-using .Distributions: Uniform, Normal
+using .Distributions: Distribution, Uniform, Normal
```

Comment:
You never used `Distribution`... :confused:

Reply:
Right, in the beginning I had
```julia
mutable struct Sampler 
    X::LazySet
    sampler::Vector{<:Distribution}
end
```
Should I add this? What is best practice?

Reply:
I think the best is to add type parameters.
```julia
struct Sampler{S<:LazySet, D<:Distribution}
    X::S
    sampler::Vector{D}
end
```
Note that the struct needs not be `mutable` if you only ever modify the `Vector`'s contents.

Reply:
> What is best practice?

Indeed, for library code it is preferable to consider using type parameters; see the discussion in https://docs.julialang.org/en/v1/manual/performance-tips/#Type-declarations-1

Reply:
Thanks for the explanation. I will add your version.